rsvim_core/src/ui/widget/command_line.rs

```diff
@@ -211,6 +242,51 @@ impl CommandLine {
   pub fn content_id(&self) -> TreeNodeId {
     self.content_id
   }
+
+  /// Command-line message widget ID.
+  pub fn message_id(&self) -> TreeNodeId {
+    self.message_id
+  }
+
+  /// Command-line message setter.
+  pub fn set_message(&mut self, text: Option<CompactString>) {
+    self.message_mut().set_message(text);
+  }
+
+  /// Command-line message visibility setter
+  pub fn set_message_visible(&mut self, visible: bool) {
+    self.content_mut().set_visible(!visible);
+    self.message_mut().set_visible(visible);
+    let message_contents = self
+      .message_mut()
+      .get_text_contents_mut()
+      .upgrade()
+      .unwrap();
+    let mut message_contents = lock!(message_contents);
+    message_contents.command_line_content_mut().clear();
+    if !visible {
+      self.message_mut().set_message(None);
+    }
+  }
+
+  /// Command-line message visibility getter
+  pub fn get_message_visible(&self) -> bool {
+    self.message().visible()
+  }
```

Comment:
`get_message_visible()` this method is too simple, we can remove it, and let the caller directly call `message().visible()`.